mne/bem.py

```diff
@@ -1108,12 +1107,112 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
 
     # Show computed BEM surfaces
     if show:
-        plot_bem(subject=subject, subjects_dir=subjects_dir,
+        plot_bem(subject=subject, sub1jects_dir=subjects_dir,
                  orientation='coronal', slices=None, show=True)
 
     logger.info('Created %s\n\nComplete.' % (fname_head,))
 
 
+def convert_surface(fname_in, fname_out, T1_mgz, useLAS=False):
+    """Convert surface.
+
+    Parameters
+    ----------
+    fname_in : str
+        Path to surface in FreeSurfer surf format.
```

Comment:
and this should probably take a `rr, tris` combo, not `fname` (user can call `read_surface` and then this calls the geom adding functions for them)